### PR TITLE
Fix build failure in Meta internal test

### PR DESCRIPTION
**Pull request by xiaoxmeng**

### CODE REVIEW

File: velox/common/memory/SharedArbitrator.cpp [hunk 1]
```diff
@@ -271,12 +271,13 @@ uint64_t SharedArbitrator::decrementFreeCapacity(
...
-  uint64_t allocatedBytes = std::min(freeNonReservedCapacity_, maxBytes);
+  uint64_t allocatedBytes =
+      std::min<uint64_t>(freeNonReservedCapacity_, maxBytes);
   VELOX_CHECK_LE(allocatedBytes, freeNonReservedCapacity_);
   freeNonReservedCapacity_ -= allocatedBytes;
   if (allocatedBytes < minBytes) {
-    const uint64_t reservedBytes =
-        std::min(minBytes - allocatedBytes, freeReservedCapacity_);
+    const int64_t reservedBytes =
+        std::min<int64_t>(minBytes - allocatedBytes, freeReservedCapacity_);
```
Comment (ggezer) [inline velox/common/memory/SharedArbitrator.cpp:280]: why not ` std::min<uint64_t>` here as well?

File: velox/common/memory/SharedArbitrator.cpp [hunk 2]
```diff
@@ -271,12 +271,13 @@ uint64_t SharedArbitrator::decrementFreeCapacity(
...
     uint64_t minBytes) {
-  uint64_t allocatedBytes = std::min(freeNonReservedCapacity_, maxBytes);
+  uint64_t allocatedBytes =
+      std::min<uint64_t>(freeNonReservedCapacity_, maxBytes);
   VELOX_CHECK_LE(allocatedBytes, freeNonReservedCapacity_);
   freeNonReservedCapacity_ -= allocatedBytes;
   if (allocatedBytes < minBytes) {
-    const uint64_t reservedBytes =
-        std::min(minBytes - allocatedBytes, freeReservedCapacity_);
+    const int64_t reservedBytes =
```
Comment (ggezer) [inline velox/common/memory/SharedArbitrator.cpp:279]:
```suggestion
    const uint64_t reservedBytes =
```

**Review by ggezer — APPROVED** [review]
LGTM. Thanks for a quick fix.

**Review by amitkdutta — APPROVED** [review]
Thanks @xiaoxmeng

**Pull request closed without merging**